src/game.cpp

```diff
@@ -7978,6 +7985,15 @@ bool game::npc_menu( npc &who )
             u.melee_attack( who, true );
             who.make_angry();
         }
+    } else if( choice == disarm ) {
+        if( !who.is_enemy()) {
+            if( !query_yn( _("You may be attacked! Proceed?") ) ) {
```

Comment:
Minor: can be shortened to `if ( !who.is_enemy() && !query_yn( _( "You may be attacked! Proceed?" ) ) )`.
Also, style flaws (lack of spaces).